index.js

```diff
@@ -66,7 +66,8 @@ class Response {
   }
 
   setHeaders (obj) {
-    Object.keys(obj).map(key => {
+    const names = Object.keys(obj)
+    names.forEach(key => {
       this.setHeader(key, obj[key])
     })
```

Comment:
@mafintosh please check agin 👍

Reply:
Ah sorry. This is fine, but there is a conflict with index.js.